packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth/methods/sendTransaction.ts

```diff
@@ -135,43 +135,47 @@ describe("Eth module", function () {
+                const firstBlock = await getFirstBlock();
+                const hash = await this.provider.send("eth_sendTransaction", [
+                  {
+                    from: DEFAULT_ACCOUNTS_ADDRESSES[0],
+                    data: "0x00",
+                  },
+                ]);
 
-            const receipt = await this.provider.send(
-              "eth_getTransactionReceipt",
-              [hash]
-            );
+                const receipt = await this.provider.send(
+                  "eth_getTransactionReceipt",
+                  [hash]
+                );
 
-            const receiptFromGeth = {
-              blockHash:
-                "0x01490da2af913e9a868430b7b4c5060fc29cbdb1692bb91d3c72c734acd73bc8",
-              blockNumber: "0x6",
-              contractAddress: "0x6ea84fcbef576d66896dc2c32e139b60e641170c",
-              cumulativeGasUsed: "0xcf0c",
-              from: "0xda4585f6e68ed1cdfdad44a08dbe3979ec74ad8f",
-              gasUsed: "0xcf0c",
-              logs: [],
-              logsBloom:
-                "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-              status: "0x1",
-              to: null,
-              transactionHash:
-                "0xbd24cbe9c1633b98e61d93619230341141d2cff49470ed6afa739cee057fd0aa",
-              transactionIndex: "0x0",
-            };
+                const receiptFromGeth = {
+                  blockHash:
+                    "0x01490da2af913e9a868430b7b4c5060fc29cbdb1692bb91d3c72c734acd73bc8",
+                  blockNumber: "0x6",
+                  contractAddress: "0x6ea84fcbef576d66896dc2c32e139b60e641170c",
+                  cumulativeGasUsed: "0xcf0c",
+                  from: "0xda4585f6e68ed1cdfdad44a08dbe3979ec74ad8f",
+                  gasUsed: "0xcf0c",
+                  logs: [],
+                  logsBloom:
+                    "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+                  status: "0x1",
+                  to: null,
```

Comment:
Shouldn't you use `toValue` here?

Reply:
:facepalm: :pray: 

Note, though, that it shouldn't be used here, because this is the receipt; rather, it should be used further up, in the RPC call.  Regardless, :facepalm: for not even using `toValue`!

Addressed in https://github.com/nomiclabs/hardhat/pull/1524/commits/f11a9f6d705b0463df3afeeda1491e324fd69f56

Reply:
We really need a `no-unused-vars` linter warning, but we won't have one until we migrate to eslint :slightly_frowning_face: